atlasdb-feign/build.gradle

```diff
@@ -28,6 +28,7 @@ dependencies {
   explicitShadow project(':lock-api')
   explicitShadow group: 'com.squareup.okhttp3', name: 'okhttp'
   explicitShadow group: 'com.palantir.remoting-api', name: 'service-config'
+  explicitShadow group: 'com.palantir.remoting3', name: 'jaxrs-clients'
```

Comment:
I want to revisit if we need to shadow (and generally how much shadowing we need to do), but that's out of scope for this PR.